vehicle/external_cmd_converter/include/external_cmd_converter/node.hpp

```diff
@@ -38,7 +40,8 @@ using ExternalControlCommand = tier4_external_api_msgs::msg::ControlCommandStamp
 using Odometry = nav_msgs::msg::Odometry;
 using raw_vehicle_cmd_converter::AccelMap;
 using raw_vehicle_cmd_converter::BrakeMap;
-using Odometry = nav_msgs::msg::Odometry;
+using ControlCommandStamped = autoware_auto_control_msgs::msg::AckermannControlCommand;
+using GateMode = tier4_control_msgs::msg::GateMode;
```

Comment:
Do we need this?

Reply:
I think I should use the GateMode value to omit [tier4_control_msgs::msg::], but I forgot to use it. I will apply it now!